README.md

```diff
@@ -1,56 +1,37 @@
 # `ayb`
-With `ayb`, all your base can finally belong to you. Move SQL for great justice.
+Ayb is the easiest way to get started writing shareable reproducable database demo apps.  With only 5 commands you can have a sharable database.
 
 [![Build status](https://github.com/marcua/ayb/actions/workflows/tests.yml/badge.svg)](https://github.com/marcua/ayb/actions/workflows/tests.yml)
 
 
 ## Introduction
 
-`ayb` is a multi-tenant database management system with easy-to-host instances that enable you to quickly register an account, create databases, share them with collaborators, and query them from a web application or the command line. An `ayb` server allows users to create SQLite databases (other databases to come), and then exposes those databases through an HTTP API.
+`ayb` is a user-installable? database management system with easy-to-host instances that enable you to quickly register an account, create databases, share them with collaborators, and query them from a web application or the command line. An `ayb` server allows users to create SQLite databases (other databases to come), and then exposes those databases through an HTTP API.
 
 To learn more about why `ayb` matters, how it works, or who it's for, [read this introductory blog post](https://blog.marcua.net/2023/06/25/ayb-a-multi-tenant-database-that-helps-you-own-your-data.html).
 
 *alpha warning*: `ayb` is neither feature complete nor production-ready. Functionality like authentication, permissions, collaboration, isolation, high availability, and transaction support are on the [Roadmap](#roadmap) but not available today. I work on `ayb` as a hobbyist side project.
 
 ## Getting started
 
-### Installing
-`ayb` is written in Rust, and is available as the `ayb` crate. Assuming you have [installed Rust on your machine](https://www.rust-lang.org/tools/install), installing `ayb` takes a single command:
-
-```bash
-cargo install ayb
-```
-
-### Running a server
-An `ayb` server stores its metadata in [SQLite](https://www.sqlite.org/index.html) or [PostgreSQL](https://www.postgresql.org/), and stores the databases it's hosting on a local disk. To configure the server, create an `ayb.toml` configuration file to tell the server what host/port to listen for connections on, how to connect to the database, and the data path for the hosted databases:
-
+### Quickstart 
 ```bash
-$ cat ayb.toml
```

Comment:
possibly replace this `cat ayb.toml` with an `ayb show_config` command.  A developer knows what `cat` does, a student might not.

Reply:
Great suggestion! Added an issue at https://github.com/marcua/ayb/issues/142